consume.go

```diff
@@ -115,43 +119,45 @@ func (c *Consumer) startConsuming() error {
 	}
 
 	for i := 0; i < c.options.HandlerQuantity; i++ {
-		go c.handlerRoutine(deliveries, c.options, c.handler)
+		go c.handlerRoutine(deliveries, c.options)
 	}
 
 	c.conn.logger.logDebug(fmt.Sprintf("Processing messages on %d message handlers", c.options.HandlerQuantity))
 
 	return nil
 }
 
-func (c *Consumer) handlerRoutine(deliveries <-chan amqp.Delivery, consumeOptions *ConsumeOptions, handler HandlerFunc) {
-	for msg := range deliveries {
+func (c *Consumer) handlerRoutine(deliveries <-chan amqp.Delivery, consumeOptions *ConsumeOptions) {
```

Comment:
We could remove the parameter `consumeOptions` and use it from the receiver `c *Consumer` instead